fastify.js

```diff
@@ -356,6 +388,9 @@ function fastify (options) {
     if (name === 'onClose') {
       this[kHooks].validate(name, fn)
       this.onClose(fn)
+    } else if (name === 'onReady') {
+      this[kHooks].validate(name, fn)
+      this[kHooks].add(name, fn)
```

Comment:
Which version is correct? The one you have here? Or the one I have in PR #2191?:

https://github.com/fastify/fastify/blob/eca2bbc20a4f44ad84db66c8a092381ec99bb4f1/fastify.js#L404-L407

Reply:
In `next` the global hooks have been removed since `onRoute` and `onRegister` has been encapsulated.
I kept the same logic for this hook, but it would be good as global hooks in `v2` since all the functions added are called once.

The only difference from this PR and that one is that the `this` context is not binded to the instance where the `addHook` is called 👍